examples/screenshot_and_export_figure.py

```diff
@@ -0,0 +1,158 @@
+"""
+Comparison of Screenshot and Figure Export
+=============
```

Comment:
```suggestion
Comparison of Screenshot and Figure Export
==================================
```

the underline needs to be the same length as the title, see:
https://app.circleci.com/pipelines/github/napari/napari/7084/workflows/3ee584eb-a41d-4765-a4c1-6f275231e443/jobs/6889?invite=true#step-106-94366_95

Reply:
is there a way for either my linter or pre-commit to tell me this? Or is this just something I need to know. Perhaps it already should and I don't have something set up correctly.